D/Display/DisplayController.cs

```diff
@@ -171,7 +171,7 @@ private void HorizontalRetraceCallback(ulong skewNsec, object context)
                 {
                     // Normal line : 32 bits of border pattern, 1024 bits of display, 32 bits of border pattern
                     // Border pattern: low byte on lines 4n, 4n+1; high byte on 4n+2, 4n+3.
-                    int patternByte = (effectiveScanline & 0x2) == 0 ? _displayBorder & 0xff : _displayBorder >> 8;
+                    int patternByte = (effectiveScanline & 0x2) == 2 ? _displayBorder & 0xff : _displayBorder >> 8;
```

Comment:
The root cause of this issue is more subtle.  Without this change, Interlisp-D's border pattern aligns properly and Viewpoint is misaligned; with this change the opposite will be true.  I suspect the root cause is a timing issue during the start of the frame that I have to dig into.  I'd prefer not to change this right now.  (See also line 216.)